docs/src/_includes/layouts/doc.html

```diff
@@ -148,4 +148,27 @@ <h1>{{ title }}</h1>
     </a>
 </div>
 
+<div id="copy-announcement" class="visually-hidden" aria-live="polite" aria-atomic="true"></div>
+
+<script>
+    (function() {
+        const copyBtns = document.querySelectorAll("#copyBtn");
```

Comment:
```suggestion
        const copyBtns = document.querySelectorAll(".copy-btn");
```
Let's use class names instead of IDs to identify buttons.